projects/fw102_example/src/main.c

```diff
@@ -0,0 +1,51 @@
+//
+// Created by Matthew Keller on 2022-01-22.
+//
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "delay.h"       // For real-time delays.
+#include "interrupt.h"   // For interrupting stuff
+#include "log.h"         // For printing stuff.
+#include "soft_timer.h"  // Software timers for scheduling future events.
+#include "wait.h"  //
+
+#include "gpio.h"  // General Purpose I/O control.
+#include "misc.h"  // Various helper functions/macros.
+
+#define COUNTER_LENGTH 500
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void increment_a(SoftTimerId timer_id, void *context) {
```

Comment:
Functions that are only used in one file should use the static label and be prefixed with prv_, (ie. `static void prv_increment_a`). This lets us know that it is limited to the scope of the file that it is in